arex-storage-web-api/src/main/java/com/arextest/storage/metric/MetricListener.java

```diff
@@ -17,4 +17,9 @@ public interface MetricListener {
    * record match strategy count with tags.
    */
   void recordMatchingCount(String metricName, Map<String, String> tags);
+
+  /**
+   * record request and response size with tags.
+   */
+  void recordRequestAndResponseSize(String metricName, Map<String, String> tags, long requestSize, long responseSize);
```

Comment:
Here you need a recordSize method

`void recordSize(String metricName, Map<String, String> tags, long timeMillis);`